Modify migration documents to download migration artifacts (#6125)
wso2/docs-apim: en/docs/install-and-setup/upgrading-wso2-api-manager/300-to-410/upgrading-from-300-to-410.md

You have to run the following migration client to update the API Manager artifacts.

1. Download and extract the [migration-resources.zip]({{base_path}}/assets/attachments/install-and-setup/migration/migration-resources.zip). Copy the extracted migration-resources to the `<API-M_4.1.0_HOME>` folder.

2. Copy the [org.wso2.carbon.apimgt.migrate.client-4.1.0.x.jar]({{base_path}}/assets/attachments/install-and-setup/migration/org.wso2.carbon.apimgt.migrate.client-4.1.0.31.jar) file into the `<API-M_4.1.0_HOME>/repository/components/dropins` directory.

> 3. Prior to API-M migration, run the below commad to execute pre migration step which will validate your old data.

Reviewer: ```suggestion
3. Prior to API-M migration, run the below command to execute the pre-migration step that will validate your old data.
```

    -DrunPreMigration
    ```

>     !!! note "If you want to save the invalid api definitions"

Reviewer: ```suggestion
    !!! note "If you want to save the invalid API definitions"
```

>         You can save the invalid api definitions to the local file system during this data validation step if required. Use the `-DsaveInvalidDefinition` option for this as follows. The invalid definitions will be stored under a folder named `<API-M_4.1.0_HOME>/invalid-swagger-definitions` in the form of `<API_UUID>.json`. Then you can manually correct these definitions.

Reviewer: ```suggestion
        You can save the invalid API definitions to the local file system during this data validation step if required. Use the `-DsaveInvalidDefinition` option for this as follows. The invalid definitions will be stored under a folder named `<API-M_4.1.0_HOME>/invalid-swagger-definitions` in the form of `<API_UUID>.json`. Then you can manually correct these definitions.
```
